manifest_staging/charts/csi-secrets-store-provider-azure/values.yaml

```diff
@@ -3,6 +3,10 @@ image:
   tag: 0.0.8
   pullPolicy: IfNotPresent
 
+# One or more secrets to be used when pulling images
+imagePullSecrets: []
```

Comment:
Could we change this to 

```
# One or more secrets to be used when pulling images
# imagePullSecrets:
#   - name: myRegistryKeySecretName
```

so we can just populate the values directly in the helm chart?